### DISCUSSION TITLE
The function of `watch` a repo should be able to be customed by `tags`

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [community/community#18195] [Other Feature Feedback, Questions, & Ideas]
Now, The function of `watch` a repo can customed by issues and prs generally. I want it could be more accurate. For example, I only want   to receive the notifications that the prs and issues marked a `waiting for feedback` and etc.

**Comment by RandyMcMillan** [comment]
It would be great if we could set the default options (including watch) to a list of options - most of the time I just want to

watch: [release]

it would be great to have a be able to set your default watch options as a yaml list.

watch: [ "issues","pull-requests", "release", "discussions", "security-alerts", options: [ "mentions", ... ] ]

Users could set what ever they want their default to be. The yaml file could be included in the special repos:

https://github.com/<user_name>/<user_name>/.github/OPTIONS.yml 

https://github.com/<org_name>/.github/OPTIONS.yml

**Reply by awxiaoxian2020** [reply]
It would be a greate solution! expecting the response from GitHub.
